client/src/routes/+layout.svelte

```diff
@@ -0,0 +1,5 @@
+<script>
+  import '../app.css';
```

Comment:
絶対パスを使いたい

Reply:
以下のように書き換えたが、500 になったため一旦後回しにする
```suggestion
  import '@/app.css';
```